blocks/cards/cards.css

```diff
@@ -1,3 +1,12 @@
+:where(.cards-card-body) {
+  --category-size: var(--font--2);
+  --title-size: var(--font-2);
+  --subtitle-size: var(--font--1);
+  --description-size: var(--font--1);
+  --content-spacing: var(--space-2);
+  --content-end-spacing: var(--space-5);
```

Comment:
where are you using all these variables?

Reply:
We are using these variables in cards.css file. We might need these for other variations of card in future as well.

Reply:
I don't see them being used in this file, could you link to the specific lines?

Reply:
2 of them are used, so removed the other vars. Line 114,119